src/clicache/integration-test/CacheHelperN.cs

```diff
@@ -1877,7 +1878,7 @@ public static int getBaseLocatorPort()
           }
           try
           {
-            TextWriter tw = new StreamWriter(Directory.GetCurrentDirectory() + "\\" + startDir + "\\gemfire.properties", false);
+            TextWriter tw = new StreamWriter(Directory.GetCurrentDirectory() + "\\" + startDir + "\\geode.properties", false);
```

Comment:
This is an output properties file created by the tests and is OK to call geode.properties. Thanks for checking though.